tests/ttnn/sweep_tests/sweeps/sweeps/heaviside.py

```diff
@@ -16,14 +16,29 @@
     "batch_sizes": [(1,)],
     "height": [384, 1024],
     "width": [1024, 4096],
-    "input_dtype": [ttnn.bfloat16],
+    "input_dtype": [ttnn.bfloat16, ttnn.bfloat8_b],
     "input_memory_config": [ttnn.DRAM_MEMORY_CONFIG],
     "output_memory_config": [ttnn.DRAM_MEMORY_CONFIG],
-    "layout": [ttnn.TILE_LAYOUT],
+    "layout": [ttnn.TILE_LAYOUT, ttnn.ROW_MAJOR_LAYOUT],
     "value": [0.5, 0.6],
 }
 
 
+def skip(
+    batch_sizes,
+    height,
+    width,
+    input_dtype,
+    input_memory_config,
+    output_memory_config,
+    layout,
+    value,
+) -> Tuple[bool, Optional[str]]:
+    if layout == ttnn.ROW_MAJOR_LAYOUT or input_dtype == ttnn.bfloat8_b:
+        return True, "Skipped since ROW_MAJOR_LAYOUT is not supported and test fails for bfloat8_b"
```

Comment:
Test fail means pcc drop or something else? Juse not supported is not enough?

Reply:
Is it bf8 supported?

Reply:
It is failing (not crashing) for bf8_b. So I have updated the pybind to support only bf16.

Reply:
@umadevimcw @Aswinmcw 
bf8_b is now passing with high pic. I have updated the document accordingly.